### DISCUSSION TITLE
Radio not configured

### DISCUSSION THREAD

**Discussion by Marcovaldo78** [cedricp/ddt4all#1055] [Q&A]
Hello,
I have a Twingo III GT and after messing up my rlink to activate RS monitor (go to your dealer), I bought a spare rlink. Obviously after turning it on, "Radio not configured" message.
If I have understand, I must write with pyren correct parameters right? 
Thank you very much

Gianluca

**Comment by KarelSvo** [comment]
https://github.com/cedricp/ddt4all/discussions/967

**Comment by Marcovaldo78** [comment]
I have read all the discussions and threads about Pyren and ddt4all…I changed my rlink display and now I have "radio not configured" message… Pyren recognize my ecus but when I apply macro gaves error "service not supported in active session" . I use USB elm and I also changed parameters (min_tout)
I tried on PC with USB and wifi elm, changed USB to 230400, changed latency time
Also tried on Android phone (qpyren3)...same errors
My elm works on ddt4all and on Pyren, only macro commands gaves me those type of errors
Anyone can help me?
 Thank you

**Reply by shrlnm** [reply]
@Marcovaldo78 Please share elm_logs from your tries.
